sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -145,6 +147,38 @@ public void testCopyFromIntoPartitionedTable() throws Exception {
     }
 
     @Test
+    public void testInsertIntoClosedPartition() throws Exception {
+        execute("create table t (n integer) partitioned by (n)");
+        ensureYellow();
+        execute("insert into t (n) values (1)");
+        refresh();
```

Comment:
refresh is not needed - if you want to wait for partitions to be created you'd have to use ensureYellow/ensureGreen